docs/getting-started.md

```diff
@@ -44,74 +43,138 @@ Verify that the image is available in your account and remember the correspondin
 ```
 
-Deploy a CNI solution, Calico is used here as an example.
+After the first control plane node has initialized status, you can retrieve the workload cluster Kubeconfig:
 
 ```bash
-kubectl --kubeconfig=./${CLUSTER_NAME}.kubeconfig apply -f https://docs.projectcalico.org/v3.8/manifests/calico.yaml
+$ clusterctl get kubeconfig capdo-quickstart > capdo-quickstart.kubeconfig
 ```
 
-Deploy DigitalOcean Cloud Controller Manager
+You can verify the kubernetes node in the workload cluster
 
 ```bash
-kubectl --kubeconfig=./${CLUSTER_NAME}.kubeconfig apply -f examples/digitalocean-cloud-controller-manager.yaml
+$ KUBECONFIG=capdo-quickstart.kubeconfig kubectl get node
+
+NAME                                   STATUS     ROLES    AGE     VERSION
+capdo-quickstart-control-plane-pt926   NotReady   master   10m     v1.17.11
+capdo-quickstart-md-0-2vnwv            NotReady   <none>   5m31s   v1.17.11
+capdo-quickstart-md-0-5295f            NotReady   <none>   5m30s   v1.17.11
+capdo-quickstart-md-0-pm8np            NotReady   <none>   5m28s   v1.17.11
 ```
 
-Optional Deploy DigitalOcean CSI
+### Deploy CNI
+
+Calico is used here as an example.
 
 ```bash
-kubectl --kubeconfig=./${CLUSTER_NAME}.kubeconfig apply -f examples/digitalocean-csi.yaml
+$ KUBECONFIG=capdo-quickstart.kubeconfig kubectl apply -f https://docs.projectcalico.org/manifests/calico.yaml
 ```
 
-Check the status of control-plane using kubectl get nodes
+### Deploy DigitalOcean CCM and CSI
 
 ```bash
-kubectl --kubeconfig=./${CLUSTER_NAME}.kubeconfig get nodes
+# Create digitalocean secret
+$ KUBECONFIG=capdo-quickstart.kubeconfig kubectl create secret generic digitalocean --namespace kube-system --from-literal access-token=$DIGITALOCEAN_ACCESS_TOKEN
+
+# Deploy DigitalOcean Cloud Controller Manager
+$ KUBECONFIG=capdo-quickstart.kubeconfig kubectl apply -f https://raw.githubusercontent.com/digitalocean/digitalocean-cloud-controller-manager/master/releases/v0.1.27.yml
+
+# Deploy DigitalOcean CSI (optional)
+$ KUBECONFIG=capdo-quickstart.kubeconfig kubectl apply -f https://raw.githubusercontent.com/digitalocean/csi-digitalocean/master/deploy/kubernetes/releases/csi-digitalocean-v1.3.0.yaml
 ```
 
-Finishing up, we’ll create a single node MachineDeployment.
+After CNI and CCM deployed, your workload cluster nodes will turn into ready. You can verify using:
+
+```bash
+$ KUBECONFIG=capdo-quickstart.kubeconfig kubectl get node
 
+NAME                                   STATUS   ROLES    AGE   VERSION
+capdo-quickstart-control-plane-pt926   Ready    master   25m   v1.17.11
+capdo-quickstart-md-0-2vnwv            Ready    <none>   21m   v1.17.11
+capdo-quickstart-md-0-5295f            Ready    <none>   21m   v1.17.11
+capdo-quickstart-md-0-pm8np            Ready    <none>   21m   v1.17.11
 ```
-kubectl apply -f examples/_out/machinedeployment.yaml
+
+## Deleting a workload cluster
+
+You can delete the workload cluster from management cluster using:
```

Comment:
grammar nit: `You can delete the workload cluster from management cluster using:` should be `You can delete the workload cluster from the management cluster using:`

In other words, can we update this so that the definite article `the` is used before the noun `management cluster`: 
`from management cluster` to `from the management cluster`.